packages/discovery-provider/src/tasks/entity_manager/entities/user.py

```diff
@@ -611,7 +620,32 @@ def verify_user(params: ManageEntityParameters):
     )
 
     user_record = validate_user_record(user_record)
-    user_record.is_verified = True
+
+    metadata = params.metadata
+    # todo: default below to false instead of true after identity change has been deployed
+    is_verified = metadata.get("is_verified", True)
```

Comment:
this took me a second to understand, but this is because the current tx does not contain the is_verified metadata at all right? leaving this comment in for historical sake

Reply:
yup